pom.xml

```diff
@@ -293,9 +204,6 @@
 							<goal>integration-test</goal>
 						</goals>
 						<configuration>
-							<includes>
```

Comment:
Instruction in README.md to start server in Eclipse should be

4. Right-click the project and select Run As -> Run on Server.